application/fpga/python/DobotDriver.py

```diff
@@ -495,6 +497,29 @@ def LaserOn(self, on):
 			result = self._write1(CMD_LASER_ON, 0)
 		self._lock.release()
 		return result
+	
+	def PumpOn(self, on):
+		'''
+		'''
```

Comment:
Now that I looked at the pump and the valve again I think it will be useful to have both these commands but also a third one that would enable them both. The reasoning is the following:
* the pump enabled with valve open will not do anything because the air will be sucked through the valve. However, this enables you have more precise control over when to enable/disable low pressure as the pump motor has some inertia. So by keeping the pump on you can even generate rapid bursts of lower pressure, even lower than the pump can do because of the physics of the air flow
* the third command would enable us to save a command cycle (20ms cycle), when these are properly queued along with the movement commands, as there is simply no time to execute a pump command and not miss an fpga command, unless we do command pre-fetching (looking ahead at what is the next command and execute it if it is a non-fpga command right after fpga command has been sent). However, command pre-fetching has a side effect of these commands being executed a little prematurely - 20ms ahead of planned time, though that may be not a big deal.